findmyparadise/src/main/java/com/tp/wrc/findmyparadise/controllers/PhotoGalleryController.java

```diff
@@ -1,9 +1,10 @@
 package com.tp.wrc.findmyparadise.controllers;
+import com.tp.wrc.findmyparadise.exceptions.InvalidHostIDException;
+import com.tp.wrc.findmyparadise.exceptions.NullHostIDException;
 import com.tp.wrc.findmyparadise.services.PhotoGalleryService;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.web.bind.annotation.CrossOrigin;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
 
 @RestController
 @RequestMapping("api")
```

Comment:
You should probably make the overall request mapping more specific to this controller.
Right now all the endpoint are just on `"/api"` e.g. `"/api/getById/{id}"` which will likely cause failures and collisions with the other controllers.
Try setting `@RequestMapping("api/photos")`or something like that

Reply:
I gotcha ill make those changes too